src/schema/schema.graphql

```diff
@@ -737,6 +739,11 @@ type TzOnsetInfo { # <standard>, <daylight>
 	sec: Int
 }
 
+type AppointmentInfo { # <appt>; aka AppointmentHitInfo
```

Comment:
how is this different then CalendarItemHitInfo ?

Reply:
both contains same data

Reply:
CalendarItemHitInfo having so many mandatory fields, which is not the case here

Reply:
ok